### DISCUSSION TITLE
Contribute this Integration as replacement for the yamaha core integration

### DISCUSSION THREAD

**Discussion by ChristophCaina** [mvdwetering/yamaha_ynca#145] [Ideas]
I don't see much difference between this - and the integration which is already available within the HA Core... except: this one comes with an UI based config flow and the core integration does not.

Therefore, I would suggest to consider this integration as an replacement for the core integration :)

**Comment by mvdwetering** [comment]
While I do think the integration matches most Home Assistant quality standards I will not submit this integration the Home Assistant Core.

Due to the large amount of pullrequests in HA core adding a new integration can take a _really_ long time (>6 months is not uncommon). Next to that a new integration needs to be added one platform (=entity type) at the time which is also additional work. I don't think it is worth the additional effort. 

And when it is in core I will lose flexibility in when I add/release things as everything has to be reviewed which can also take a while. Although having reviews is valuable as it will probably result in better quality code, it is a hassle I don't need.

For me this is a hobby project and maybe a bit selfish, but the time spent on this needs to be fun and the current state of the pullrequest process does not seem fun. (note that I don't think it is a bad process, it is just not for me)
